experiments/interpretation/dutchf3_patch/local/azureml_requirements.txt

```diff
@@ -0,0 +1,13 @@
+git+https://github.com/olgaliak/seismic-deeplearning.git@azureml-train-pipeline#egg=cv_lib&subdirectory=cv_lib
+git+https://github.com/microsoft/seismic-deeplearning.git#egg=deepseismic-interpretation&subdirectory=interpretation
+opencv-python==4.1.2.30
```

Comment:
![image](https://user-images.githubusercontent.com/8856246/74266093-8d81aa00-4cb8-11ea-8276-092dd495a37b.png)

@maxkazmsft

Reply:
Opened up an issue to cover migration from OpenCV to Pillow in script functionality and environment requirements #203